src/lib/connectors/repos/billing-volumes.js

```diff
@@ -131,10 +131,10 @@ const deleteByBatchIdAndLicenceId = (billingBatchId, licenceId) =>
  * @param {String} billingBatchId
  * @param {Object} changes
  */
-const updateByBatchId = async (billingBatchId, changes) => {
+const updateByBatchId = async (billingBatchId, changes, options = {}) => {
```

Comment:
As this function is only called in 1 place, we could only add `{ require: false }` to the call below rather than exposing Bookshelf options to the invoking code